src/osm_configurator/view/popups/yes_no_pop_up.py

```diff
@@ -22,6 +23,37 @@ def __init__(self, message, func):
 
         Args:
             message (str): The message to be shown in the PopUp.
-            func (typing.Callable): A Function that takes one Boolean and has no return, for the PopUp to send a message back.
+            func (typing.Callable): Function taking one Boolean and has no return for the PopUp to return a message
         """
-        pass
+        self.func = func
+
+        popup = super().__init__()
+        self.geometry(POPUPSIZE)
+
+        self.title("Alert")
+
+        customtkinter.CTkLabel(popup, text=message) \
+            .pack(side="top", fill="both", expand="True", padx=10, pady=10)
+
+        customtkinter.CTkButton(self, text="Accept", command=combine_funcs(accept, self.destroy)) \
```

Comment:
the 'accept' might need a self. to work properly

Reply:
Changed in next commit